src/main/java/de/ellpeck/rockbottom/world/tile/TileLog.java

```diff
@@ -48,6 +48,12 @@ public void onDestroyed(IWorld world, int x, int y, Entity destroyer, TileLayer
         }
     }
 
+    public float getTranslucentModifier(IWorld world, int x, int y, TileLayer layer, boolean skylight) {
+        if (world.getState(layer, x, y).get(StaticTileProps.LOG_VARIANT).isNatural())
+            return skylight ? 1F : layer == TileLayer.BACKGROUND ? 0.9f : 0.8f;
```

Comment:
no. separate the conditions

Reply:
u literally watched me type this but ok

Reply:
yes, and both myself and ellpeck clearly laughed at you and said to change it :P